docs/governance/how-is-etherlink-governed.md

```diff
@@ -38,14 +54,20 @@ To pass, the proposal must meet both of these requirements:
 - Supermajority: The total voting power of the Yea votes must reach a supermajority.
 
 The thresholds for these requirements are stored in the governance contract.
+This table shows the requirements as of the Bifröst Etherlink update:
+
+Requirement | Threshold
+--- | ---
+Quorum | 5% of all voting power must vote Yea, Nay, or Pass
+Supermajority | 75% of Yea or Nay votes must be Yea
```

Comment:
Why not the same format as other contracts 
```
Period | Requirement | Threshold
--- | --- | ---
Proposal | Quorum | 1% of all voting power must vote for a specific proposal
Promotion | Quorum | 5% of all voting power must vote Yea, Nay, or Pass
Promotion | Supermajority | 80% of Yea or Nay votes must be Yea
```

Reply:
![image](https://github.com/user-attachments/assets/f30ddd55-e21b-44a9-b554-25ae71f06bd4)

Reply:
Because for the kernel section there is a separate section for each period, so this section is just for the promotion period. Not sure if it makes sense to do every section the same or the way it is (full info about kernel governance, other sections a summary of the changes).